src/core/components/SurveyNotice/index.js

```diff
@@ -73,34 +92,58 @@ export const SurveyNoticeBase = ({
     });
   };
 
-  // Pass along a source derived from the current URL path but with
-  // the preceding language path removed.
-  const source = querystring.stringify({
-    source: location.pathname
-      .split('/')
-      .slice(2)
-      .join('/'),
-  });
-  const surveyUrl = `https://qsurvey.mozilla.com/s3/addons-mozilla-org-survey?${source}`;
-
-  return (
-    <Notice
-      actionHref={surveyUrl}
-      actionOnClick={onDismiss}
-      actionTarget="_blank"
-      actionText={i18n.gettext('Take short survey')}
-      className="SurveyNotice"
-      dismissible
-      id="amo-experience-survey"
-      onDismiss={onDismiss}
-      type="generic"
-    >
-      {i18n.gettext(
-        'Thanks for visiting this site! Please take a minute or two to tell Firefox about your experience.',
-      )}
-    </Notice>
-  );
-};
+  onDismiss = () => {
+    this.dismissNotice();
+    this.props._tracking.sendEvent({
+      action: SURVEY_ACTION_DISMISSED,
+      category: SURVEY_CATEGORY,
+    });
+  };
+
+  onClickSurveyLink = () => {
+    this.dismissNotice();
+    this.props._tracking.sendEvent({
+      action: SURVEY_ACTION_VISITED,
+      category: SURVEY_CATEGORY,
+    });
+  };
+
+  render() {
+    const { i18n, location } = this.props;
+
+    if (!this.shouldShowNotice()) {
+      return null;
+    }
+
+    // Pass along a source derived from the current URL path but with
+    // the preceding language path removed.
+    const source = querystring.stringify({
+      source: location.pathname
+        .split('/')
+        .slice(2)
+        .join('/'),
+    });
+    const surveyUrl = `https://qsurvey.mozilla.com/s3/addons-mozilla-org-survey?${source}`;
```

Comment:
We have a `addQueryParams()` helper too in `core/utils`. I am wondering if that would do the trick here.